ext/storage_sources/local_store/local_store.c

```diff
@@ -777,6 +777,7 @@ local_location_list_internal(WT_STORAGE_SOURCE *storage_source, WT_SESSION *sess
     *countp = count;
 
 err:
+    free(dirp);
```

Comment:
This memory comes out of `opendir` and is malloc'd under the hood. Need to free this on the way out.